src/server.js

```diff
@@ -128,7 +128,9 @@ app.get('*', (req, res) => {
         // $FlowFixMe: isn't an issue
         const webpackManifest = require('../public/webpack-assets.json');
         assets = bundles
-          .map(({ publicPath }) => publicPath)
+          .map(({ publicPath }) =>
+            !publicPath.includes('main') ? publicPath : ''
```

Comment:
@wellyshen I suppose, we need this check, because it is generated #220, when `.css` and `.js` files twice rendered on DOM for production. I'm not quite sure, but I think it happened because in the file `loadable-assets.json` there are the same `publicPath` as in the `webpack-assets.json`.

<img width="931" alt="Screen Shot 2019-04-09 at 18 54 43" src="https://user-images.githubusercontent.com/12545211/55816273-9171f800-5afa-11e9-99b9-8d5c950a4c2f.png">

Reply:
@aslapkov Can you reuse the same map function like :

```js
const mapFn = ({ publicPath }) => !publicPath.includes('main') ? publicPath : '';
let assets = bundles
        .map(mapFn)
        // In development, main.css and main.js are webpack default file bundling name
        // we put these files into assets with publicPath
        .concat(['/assets/main.css', '/assets/main.js']);

      if (!__DEV__) {
        // $FlowFixMe: isn't an issue
        const webpackManifest = require('../public/webpack-assets.json');
        assets = bundles
          .map(mapFn)
          .concat(
            Object.keys(webpackManifest)
              .map(key => webpackManifest[key])
              .reverse()
          );
      }
```

Reply:
@wellyshen Of course, this is a cleaner solution!